xml/docu_styleguide.structure.xml

```diff
@@ -647,6 +647,24 @@ data  home  lost+found  opt     run   srv      tmp&lt;/screen&gt;</screen>
       </listitem>
      </itemizedlist>
     </listitem>
+    <listitem>
+      <para>To enable automatic syntax highlighting, add a language 
```

Comment:
I'm not sure if this is needed (or it's clear from the context), but "language" can be misleading. Some readers might assume we talk about _natural languages_. However, in this context, it's about _programming languages_ or _formats_.

If you want it to make it clear, here is my idea:

```suggestion
      <para>To enable automatic syntax highlighting for programming languages or formats, add a language 
```

Maybe you find a better expression.